### PR TITLE
bad sonarqube example for blog

### PR DESCRIPTION
**Pull request by sadovnikov**

### CODE REVIEW

File: minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java [hunk 1]
```diff
@@ -53,6 +53,16 @@ public AbstractContainer(MesosCluster cluster, String uuid, String containerId,
     }
 
     /**
+     * Returns lenght of the give statement
+     *
+     * @param claim string to check the length
+     * @return
+     */
+    private int getLengthOrNpe(String statement) {
```
Comment (containersol) [inline minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java]: MINOR Make "getLengthOrNpe" a "static" method. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS2325)
MAJOR Private method 'getLengthOrNpe' is never used. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by containersol** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java [hunk 2]
```diff
@@ -52,6 +52,10 @@ public AbstractContainer(MesosCluster cluster, String uuid, String containerId,
         this.config = config;
     }
 
+    private int getSystemPropertyLength(String property) {
```
Comment (containersol) [inline minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java]: MINOR Make "getSystemPropertyLength" a "static" method. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS2325)
MAJOR Private method 'getSystemPropertyLength' is never used. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by containersol** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java (hunk 2 above)
Comment (jackass0528) [inline minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java]: class Utilities {
  private static String magicWord = "magic";

  private String getMagicWord() { // Noncompliant
    return magicWord;
  }

  private void setMagicWord(String value) { // Noncompliant
    magicWord = value;
  }

}


This my guide code? And edit this code and tiling you how many code I change.

**Comment by containersol** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by sadovnikov** [comment]
Reject

**Pull request closed without merging**